package.json

```diff
@@ -12,15 +12,32 @@
     "url": "https://github.com/eventualbuddha/keysim.js"
   },
   "scripts": {
-    "build": "gobble build dist -f",
-    "test": "npm run test:node && npm run test:browser",
-    "test:browser": "./node_modules/karma/bin/karma start karma.conf.js",
-    "test:node": "mocha -R spec --recursive test"
+    "build": "browserify . --standalone keysim > dist/keysim.js",
+    "test": "npm run build && npm run test:node && npm run test:node:dist && npm run test:browser",
+    "test:browser": "karma start --single-run",
+    "test:node": "mocha --require babel-register -R spec --recursive test",
+    "test:node:dist": "DIST=true mocha --require babel-register -R spec --recursive test"
   },
   "files": [
     "dist/keysim.js",
     "lib/keysim.js"
   ],
+  "browserify": {
+    "transform": [
+      "babelify"
+    ],
+    "ignore": [
+      "jsdom"
```

Comment:
@eventualbuddha you wrote that you'd prefer rollup over browserify. Thing is, rollup isn't specifically for browsers, like browserify is.

So, this ignore option, I don't see the equivalent of it in rollup.

This ignore feature, or anything that browserify does, it does for builds that are specifically for the browser.

Also, it doesn't have the [`browser` filed](https://github.com/substack/browserify-handbook#browser-field), which is really handy.

Since this `ignore` option is used for the test only, we *can* add rollup to the devDeps and use it to build the two builds that we provide, one for the browser and one for node.

But, I don't see much point in that, other than the exercise of a tool that you believe in and would rather work with (which is legitimate).

* The browser build, if we use rollup instead of browserify for it, I don't see much use in that, other than some excessive boilerplate that you mentioned. We can theoretically get rid of browserify but then, since we don't have the `ignore` option in rollup, we'd get `jsdom` built for the browser, if at all will run (I think it won't).
* It seems to me that the node build would not benefit from rollup at this point, when there are very little dependencies to it.